bots/tests-invoke

```diff
@@ -373,8 +373,13 @@ class PullTask(object):
 
 
         env = os.environ.copy()
-        if "TEST_DATA" in env:
-            del env["TEST_DATA"]
+        # we can only use the cache for master; branches still use the old
+        # test/images/ symlink farm which we must not destroy
+        if opts.rebase:
```

Comment:
Indeed, sorry for the late-night bogus. test/images/ also exists on master.  but `[ -d test/images/scripts ]` seems to be a good enough heuristics? [rhel-7.3.6](https://github.com/cockpit-project/cockpit/tree/rhel-7.3.6) seems to be a rebase, which already has bots/, and rhel-7.4 and rhel-7.3.5 (and earlier) seem fine with that logic.